doc/source/f2py/index.rst

```diff
@@ -34,10 +34,8 @@ replace all calls to ``f2py`` mentioned in this guide with the longer version.
 .. toctree::
    :maxdepth: 3
 
-   f2py.getting-started
    f2py-user
    f2py-reference
-   usage
    python-usage
    signature-file
    buildtools/index
```

Comment:
Actually all of these can also safely be removed (as well as the `advanced` index in the next line, but I can't do that with GitHub), since they are all part of the f2py reference guide. 

The final toc should be just
``` 
f2py-user
f2py-reference
windows/index
```

Reply:
It's done.